### ISSUE TITLE
Add aria description to utility toggle

### ISSUE THREAD

**Issue by SMaxOwok** [ManifoldScholar/manifold#2021]
>This button should have a label indicating that it is a toggle button, and should probably have more descriptive textual description in the page describing what it does

I don't think a visible label is totally appropriate for this.  I do think we can add a describedby text that says what it will do though.
